src/main/java/gov/usds/case_issues/model/CaseSnoozeSummaryFacade.java

```diff
@@ -4,19 +4,25 @@
 import java.util.List;
 import java.util.Optional;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
 import gov.usds.case_issues.db.model.CaseSnooze;
+import gov.usds.case_issues.db.model.UserInformation;
 import gov.usds.case_issues.db.model.projections.CaseSnoozeSummary;
 
 /**
- * A restricting wrapper for {@link CaseSnooze}, to prevent Jackson from automatically discovering
- * fields and accessors that we do not want to expose in an API call. Could theoretically be rendered
- * unnecessary through appropriate use of Jackson annotations, but experimentally, this resulted in
+ * A restricting wrapper for {@link CaseSnooze}, to prevent Jackson from
+ * automatically discovering fields and accessors that we do not want to expose
+ * in an API call. Could theoretically be rendered unnecessary through
+ * appropriate use of Jackson annotations, but experimentally, this resulted in
  * excessive coupling between the persistence layer and the presentation layer.
  */
 public class CaseSnoozeSummaryFacade implements CaseSnoozeSummary {
 
 	private CaseSnoozeSummary wrapped;
 	private List<NoteSummary> notes;
+	private String id;
+	private String name;
 
 	public CaseSnoozeSummaryFacade(Optional<? extends CaseSnoozeSummary> optionalWrapped) {
```

Comment:
Is this constructor still used? Seems like it probably ought not to be, but that's a different question...